packages/sitecore-jss-react/package.json

```diff
@@ -63,7 +63,7 @@
   },
   "dependencies": {
     "@sitecore-jss/sitecore-jss": "21.7.0-canary.50",
-    "deep-equal": "^2.1.0",
+    "fast-deep-equal": "3.1.3",
```

Comment:
Do we need to install types for _fast-deep-equal_ or they are not needed?
_deep-equal_ types needs to be removed: https://github.com/Sitecore/jss/pull/1719/files#diff-7b807de09392e3b43f675bec4802fa16d67f108b7624e883d8c699a8f8fbd14eL32

Reply:
Should we use the latest version instead of "fixed"?